contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc

```diff
@@ -20,9 +20,23 @@ namespace Dlb {
 Envoy::Network::ConnectionBalancerSharedPtr
 DlbConnectionBalanceFactory::createConnectionBalancerFromProto(
     const Protobuf::Message& config, Server::Configuration::FactoryContext& context) {
-  const auto dlb_config = MessageUtil::downcastAndValidate<
-      const envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb&>(
-      config, context.messageValidationVisitor());
+  const auto& typed_config =
+      dynamic_cast<const envoy::config::core::v3::TypedExtensionConfig&>(config);
+  envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb dlb_config;
+  auto status = Envoy::MessageUtil::unpackToNoThrow(typed_config.typed_config(), dlb_config);
+  if (absl::IsInternal(status)) {
```

Comment:
Should probably check for `!status.ok()`.

Reply:
Sure.